ibis/backends/clickhouse/compiler/values.py

```diff
@@ -1063,3 +1063,9 @@ def _scalar_udf(op, **kw) -> str:
 @translate_val.register(ops.AggUDF)
 def _agg_udf(op, *, where, **kw) -> str:
     return agg[op.__full_name__](*kw.values(), where=where)
+
+
+@translate_val.register(ops.DateDelta)
+@translate_val.register(ops.TimestampDelta)
```

Comment:
ClickHouse doesn't have a time type. There's probably a way to simulate the `TimeDelta` operation, but that can be done in a follow up.